tests/delete_node_terminated_container/delete_node_terminate_container_suite_test.go

```diff
@@ -64,9 +64,9 @@ var _ = Describe(testName, func() {
 			ns.WaitForOutputAndLog(step, k, "Starting", 1200)
 
 			// give the cassandra container some time to get created
-			time.Sleep(20 * time.Second)
+			time.Sleep(10 * time.Second)
```

Comment:
This test seemed to be a bit flaky and failed several times. I simplified the dc in this case to just be 1 node that the test is manipulating. During the failures, the node regained readiness and recovered from us messing with the mgmt api process while it was starting. I'm not sure if it was just the 20 seconds being too generous and the pod fully started in that time... but i figured reducing to single rack/single node and cutting the time buffer in half would give us less moving parts to investigate if it happens again.

Reply:
only this test?

Reply:
yeah because this one relies on getting a node stuck in a `starting` state, which means it has to kill the mgmt api process in a limited time window in order for it to not be able to recover.

Reply:
👍